protocol/messenger_communities.go

```diff
@@ -276,6 +276,27 @@ func (m *Messenger) updateCommunitiesActiveMembersPeriodically() {
 	}()
 }
 
+func (m *Messenger) CheckCommunitiesToUnmute(response *MessengerResponse) {
+	m.logger.Debug("watching communities to unmute")
+	communities, err := m.communitiesManager.All()
+	if err != nil {
+		m.logger.Info("err", zap.Any("Couldn't get all communities", err))
+		return
```

Comment:
Another approach I like is to return a wrapped error instead of void, and leave the logging call to upstream functions (in this case `protocol.watchChatsAndCommunitiesToUnmute`). The idea is that returning an error brings more flexibility for callers to decide what to do, whereas a void function is more _opaque_. It also goes a long way to help with testing.

Something along these lines:

```go
if err != nil {
	return fmt.Errorf("couldn't get all communities: %w", err)
}
```